src/components/carousel/_index.scss

```diff
@@ -0,0 +1,165 @@
+@use "../../scss";
+@use "~slick-carousel/slick/slick";
+
+.c-carousel {
+	@include scss.component-properties("carousel");
+
+	/* Arrows */
+	.slick-prev,
+	.slick-next {
+		cursor: pointer;
+		font-size: 0;
+		line-height: 0;
+		position: absolute;
+		top: 50%;
+		display: block;
+
+		&:hover {
+			color: transparent;
+			outline: none;
+			background: transparent;
+
+			&:before {
+				@include scss.component-properties("carousel-buttons-hover-before");
+			}
+		}
+
+		&:focus {
```

Comment:
in `&:hover` and `&:focus` the same values are being set, we can avoid duplicate properties and using CSS syntax to help `&:hover, &:focus`